### PR TITLE
docs update for centos with systemd-container

### PR DESCRIPTION
**Pull request by jperrin**
Update to the documentation to include working with systemd-container.

### CODE REVIEW

**Comment by yosifkit** [comment]
LGTM

**Comment by yosifkit** [comment]
Minus the travis build being grumpy on some markdown formatting.

File: centos/content.md [hunk 1]
```diff
@@ -26,11 +26,35 @@ By default, the CentOS containers are built using yum's `nodocs` option, which h
...
+	ENV container docker
+	CMD ["/usr/sbin/init"]
+
+Build this image:
+
+	docker build --rm -t username/systemd-httpd
+
+Run the container:
+
+	docker run --ti -v /sys/fs/cgroup:/sys/fs/cgroup:ro -p 80:80 username/systemd-httpd
```
Comment (yosifkit) [inline centos/content.md]: only one dash for `-ti`

**Comment by jperrin** [comment]
Updated to correct the typo.

File: centos/content.md [hunk 2]
```diff
@@ -26,11 +26,35 @@ By default, the CentOS containers are built using yum's `nodocs` option, which h
...
+	ENV container docker
+	CMD ["/usr/sbin/init"]
+
+Build this image:
+
+	docker build --rm -t username/systemd-httpd
+
+Run the container:
+
+	docker run -ti -v /sys/fs/cgroup:/sys/fs/cgroup:ro -p 80:80 username/systemd-httpd
```
Comment (tianon) [inline centos/content.md]: This has some trailing space that `markdownfmt` isn't too happy about. :+1:

Comment (tianon) [inline centos/content.md]: Ping?  Can you remove the trailing whitespace here so Travis/`markdownfmt` are happy? :innocent:

Comment (jperrin) [inline centos/content.md]: Whoops, yes. sorry. didn't realize this was still open.

**Comment by amaltson** [comment]
:+1: for getting this merged. I had gone down several rabbit holes until I found this.

**Pull request closed without merging**